src/ui/public/doc_viewer/doc_viewer.js

```diff
@@ -48,6 +60,13 @@ uiModules.get('kibana')
     controller: function ($scope) {
       $scope.mode = docViews.inOrder[0].name;
       $scope.docViews = docViews.byName;
+
+      $scope.onKeyUp = function onKeyUp(e, name) {
```

Comment:
Hmm, for some reason this doesn't appear to be working. Notice how when I click on it, the view changes modes successfully, but when I tab and hit enter, it doesn't. (My guess is that it probably has something to do with `$parent` scope, since `mode` is directly on the `$scope` object which always causes problems.)

![may-05-2017 16-02-44](https://cloud.githubusercontent.com/assets/1178348/25767247/792547bc-31ac-11e7-90cf-09398409bfe9.gif)